### PR TITLE
Update PPP and add 'Appendix: Effort' into DG

### PR DESCRIPTION
**Pull request by tengjianling**

### CODE REVIEW

**Review by yongmingyang — APPROVED** [review]

File: docs/team/tengjianling.md [hunk 1]
```diff
@@ -1,52 +1,52 @@
...
-- Reported bugs and suggestions for other teams in the class (examples: [1](https://github.com/AY2021S1-CS2103T-W16-3/tp/issues/236), [2](https://github.com/AY2021S1-CS2103T-W16-3/tp/issues/237))
+- **New feature:** Added the feature to allow user to be quized on the next question in the quiz.
+  - What it does: allows you to be tested on another question based on the same keywords you provided for the quiz, without having to end the quiz first.
+  - Justification: It may be troublesome for the user to have to go through the process of ending the quiz and typing the whole quiz command again if he wants to quiz himself with the same keywords, and he may get the same exact same question as before even if it is randomised. This `next` command provides a convenient way for users to continually quiz themselves.
+  - Highlights: The implementation of this enhancement was challenging as additional fields have to be created to keep track of which questions to randomise from, since we do not want the `qAndA` obtained from the `next` command to be the same as the current `qAndA` that the user is seeing.
```
Comment (joshruien) [inline docs/team/tengjianling.md]: QAndA should be caps Q

File: docs/team/tengjianling.md [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
+
+- **Enhancements to Existing features:**
+  - Changed the usage message in `edit` command to the context of questions and answers: [#41](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/41)
+  - Renamed all uses of `QuestionSet` and `questionSet` to `QAndA` and `qAndA` to avoid confusion between sets and lists: [#43](https://github.com/AY2021S1-CS2103T-W15-1/tp/pull/43)
```
Comment (joshruien) [inline docs/team/tengjianling.md:32]: same here

**Review by joshruien — CHANGES_REQUESTED** [review]

File: docs/team/tengjianling.md (hunk 2 above)
Comment (tengjianling) [reply docs/team/tengjianling.md:32]: I meant renaming instance names to qAndA here

Comment (joshruien) [reply docs/team/tengjianling.md:32]: sorry did not see that.

**Review by joshruien — APPROVED** [review]

**Pull request merged**